src/main/java/edu/escuelaing/ieti/matchmaking/exception/UserMatchmakingException.java

```diff
@@ -0,0 +1,13 @@
+package edu.escuelaing.ieti.matchmaking.exception;
+
+public class UserMatchmakingException extends MatchmakingException{
```

Comment:
Given that this exception is already in a subpackage named `matchmaking`, I think it is enough to name it `UserException`